website/docs/topics/groupchat/transform_messages_speaker_selection.mdx

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
I think this and the resuming_groupchat can be removed from the commit, they are generated as needed.